tpp-app/src/home/components/LoadingVisual.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { ActivityIndicator, StyleSheet, Text, View } from "react-native";
+
+/**
+ * 
```

Comment:
One sec it's missing a wholistic function description LOL it's just a return description